# A truss containing a list of tomatoes
Tomato[] tomatoes
geometry_msgs/PoseStamped cage_location
Peduncle peduncle
